marklogic-data-hub/src/main/resources/hub-internal-config/security/amps/amps-dhf-delete-jobs.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can you also name the file "deleteJobs.json" for consistency with the other amps?